underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -209,9 +209,12 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri,
     AmazonS3ClientBuilder clientBuilder = AmazonS3ClientBuilder
         .standard()
         .withCredentials(credentials)
-        .withClientConfiguration(clientConf)
-        .withPathStyleAccessEnabled(Boolean.parseBoolean(conf
-            .get(PropertyKey.UNDERFS_S3_DISABLE_DNS_BUCKETS)));
+        .withClientConfiguration(clientConf);
+
+    if (conf.isSet(PropertyKey.UNDERFS_S3_DISABLE_DNS_BUCKETS)) {
```

Comment:
It'll always be set because it has a default value defined. So this if statement is redundant.
Earlier we were calling `withPathStyleAccessEnabled` only if it was set and true.